assets/terraform-modules/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -262,6 +262,18 @@ storage:
           kind: KubeletConfiguration
           cgroupDriver: "$${docker_cgroup_driver}"
           EOF
+    - path: /etc/docker/daemon.json
+      filesystem: root
+      mode: 0500
+      contents:
+        inline: |
+          {
+            "live-restore": true,
```

Comment:
I was reading about this in the [Docker documentation](https://docs.docker.com/config/containers/live-restore/) and it seems to not be supported for upgrading Major releases. However, AFAIU a Docker upgrade would happen by rebooting, so I guess we're fine here. Just wanted to make that explicit with this comment, no action needed :).